### ISSUE TITLE
Request for permission to use the code in my project

### ISSUE THREAD

**Issue by Gowtham-rajini** [AsmrProg-YT/Modern-Login#8]
Hi,

I hope you're doing well. I'm working on a personal project where I'm developing a login page for my website. I found your Modern-Login repository extremely helpful and have incorporated some of its features into my project. I've added several new functionalities, including a button to show the password, and have made various textual and design modifications.

As a solo developer, I would like your permission to use and modify your code for my website. 
Thank you for your time and consideration.

Best regards,
Gowtham Raj V.B.S

**Comment by Yaashiniy** [comment]
> Hi,
> 
> I hope you're doing well. I'm working on a personal project where I'm developing a login page for my website. I found your Modern-Login repository extremely helpful and have incorporated some of its features into my project. I've added several new functionalities, including a button to show the password, and have made various textual and design modifications.
> 
> As a solo developer, I would like your permission to use and modify your code for my website. Thank you for your time and consideration.
> 
> Best regards, Gowtham Raj V.B.S

Yes sure, hope my messy code doesn't give extra stress to you, XD.
